core/src/main/java/com/sequenceiq/cloudbreak/converter/PlatformResourceRequestJsonToPlatformResourceRequest.java

```diff
@@ -22,20 +25,21 @@
     @Inject
     private OrganizationService organizationService;
 
+    @Inject
+    private UserService userService;
+
     @Inject
     private RestRequestThreadLocalService restRequestThreadLocalService;
 
     @Override
     public PlatformResourceRequest convert(PlatformResourceRequestJson source) {
         PlatformResourceRequest platformResourceRequest = new PlatformResourceRequest();
+        User user = userService.getOrCreate(restRequestThreadLocalService.getIdentityUser());
+        Organization organization = organizationService.get(restRequestThreadLocalService.getRequestedOrgId(), user);
```

Comment:
these 2 lines should move to restRequestThreadLocalService. e.g: restRequestThreadLocalService.getOrganization()